src/common/config/info.js

```diff
@@ -5,6 +5,27 @@
 import { defaults as nrDefaults, getNREUMInitializedAgent } from '../window/nreum'
 import { getModeledObject } from './configurable'
 
+/**
+ * @typedef {Object} Info
+ * @property {string} [beacon]
+ * @property {string} [errorBeacon] - Base URL endpoint for all data harvested by the agent. Proxies should be defined in the init instead.
+ * @property {string} licenseKey - New Relic license key provided by the website in user account.
+ * @property {string} applicationID - New Relic application ID provided when creating a browser entity in the UI.
+ * @property {number} [sa]
+ * @property {number} [queueTime]
+ * @property {number} [applicationTime]
+ * @property {string} [ttGuid]
+ * @property {string} [user]
+ * @property {string} [account]
+ * @property {string} [product]
+ * @property {string} [extra]
+ * @property {Object} [jsAttributes] - Custom attributes that are added to majority of agent's payloads. The `setCustomAttribute` API method affects this.
+ * @property {string} [userAttributes]
+ * @property {string} [atts]
+ * @property {string} [transactionName]
+ * @property {string} [tNamePlain]
+ */
```

Comment:
same goes for these, would be helpful to mark the optional ones vs required

Reply:
Everything aside form licensekey & applicationId already marked as optional